html/pfappserver/root/static.alt/src/globals/pfConfiguration.js

```diff
@@ -2504,29 +2527,129 @@ export const pfConfigurationRoleViewFields = (args = {}) => {
       ]
     },
     {
-      label: i18n.t('Description'),
+      label: i18n.t('Name'),
       fields: [
         {
-          key: 'notes',
+          key: 'name',
           component: pfFormInput
         }
       ]
     },
     {
-      label: i18n.t('Max nodes per user'),
+      label: i18n.t('Description'),
+      fields: [
+        {
+          key: 'description',
+          component: pfFormInput,
+        }
+      ]
+    },
+    {
+      label: i18n.t('Price'), // TODO - validate the format is money
       fields: [
         {
-          key: 'max_nodes_per_pid',
+          key: 'price',
+          component: pfFormInput,
+          validators: {
+            [i18n.t('Price required')]: required,
+            [i18n.t('Enter a valid positive price')]: isAmount
```

Comment:
Split the validation for `amount` and `positive` into 2 different validators so we can reuse `isAmount` for negative values. Use `minValue(0)` for absolute values: https://monterail.github.io/vuelidate/#sub-builtin-validators